### DISCUSSION TITLE
Can this be used as an analogue to 3CX SBC for a FreePBX/other Asterisk PBX?

### DISCUSSION THREAD

**Discussion by tomck** [hnimminh/libresbc#167] [Q&A]
3CX is a commercial PBX which has one handy feature that I can't seem to find duplicated for OSS options like FreePBX, and that is the [SBC on an SBC](https://www.3cx.com/blog/docs/3cx-session-border-controller/). When you have multiple phones behind one NATted IP, it seems STUN starts to fail pretty quickly, and they removed STUN in v20 in favor of purely router phones (a phone that acts as an SBC) and the SBC software that can run on any PC or a Pi.  This works well for instances where I am using 3CX.

Something with this functionality, using a low power low heat low noise device as an SBC would be a huge benefit.

**Comment by hnimminh** [comment]
What are you trying to post this here? Advertise for 3CX? @tomck

**Reply by tomck** [reply]
I thought it was very clear? I want a session border controller that I can run on a raspberry pi and pipe all the phones through a remote branch through that instead of having them all connect directly to a FreePBX server, and I was giving an example of another existing product that does that but only for their proprietary system.

**Reply by hnimminh** [reply]
it's not clear until you state :)

**Reply by tomck** [reply]
Can this be re opened or are you saying no?
